Dockerfile

```diff
@@ -1,12 +1,13 @@
-FROM almalinux:8
+FROM fedora:34
```

Comment:
Python modulemd bindings for almalinux 8 has python3.6 version, we can bump python version for project to python3.9 (without any changes), but we can't downgrade it to python3.6
It's not affecting our app, since it's in virtualenv anyway.